RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -465,6 +465,14 @@ void borisWastedMP();
 boolean borisAcquireHP(int goal);
 boolean LM_boris();
 
+########################################################################################################
+//Defined in autoscend/paths/avatar_of_jarlsberg.ash
+boolean isJarlsberg();
```

Comment:
we typically use the format of `in_jarlsberg()` to indicate paths. so maybe do a find and rename on this?

Reply:
Personally I think this is fine. I created `isActuallyEd()` for that path. Where there's unambiguously no difference between the path and the class this should be acceptable. Also generally for coding standards, `is_something()` is used since it's easier to parse.

Reply:
fair enough